src/tree/mod.rs

```diff
@@ -197,8 +198,14 @@ impl RatchetTree {
             return unmerged_leaves;
         }
 
-        let mut left = self.resolve(treemath::left(index));
-        let right = self.resolve(treemath::right(index, size));
+        let mut left = self.resolve(treemath::left(index).expect(
+            "resolve:
+            TreeMath error when computing left child.",
```

Comment:
I'm not sure there should be line breaks in strings

Reply:
Strange. `rustfmt` didn't complain. Maybe it was respecting the newlines in my strings? Anyway. Removed the explicit line breaks and let `rustfmt` do its job.